etc/parameters/framework.json

```diff
@@ -3,6 +3,7 @@
   "hardware_parameters": "etc/parameters/hardware.json",
   "parameters_directory": "etc/parameters",
   "recording_intervals": {
-    "Control": 1
+    "Control": 1,
+    "ObjectDetectionTop": 1
```

Comment:
Wait a second... This enables recording of `ObjectDetectionTop` for every cycle. And every cycle for the entire game means, we record every image of the vision top cam?

@h3ndrk @knoellle

Reply:
If the object detection cycler runs with lower frequency, it will skip images, so not every image. I don't know, how much slower it actually runs.

Maybe this should instead be added to pepsi pregame instead of the default `framework.json`?

Reply:
But it only runs slower in Initial. In all the other gamestates, it completes instantly, as it is skipping computation in the nodes. This leads to the object detection cycler recording every image. Doesn't it?

Reply:
I didn't realize this until now tbh, but you are correct. Is there a way to completely disable a cycler at runtime?

Reply:
no

Reply:
unless you block its execution in the setup node... but that is also not a good idea

Reply:
My long term idea was to also add a way from reporting from robotics domain whether and what and how often you want to record. This way, we could disable general recording in Initial but the object detection cycler could override that if it actually runs. See https://github.com/HULKs/hulk/issues/778

Reply:
I think the general problem or too simple current state is that we are not able to express the full decision matrix currently for all orthogonal inputs/reasons/causes. These inputs are (hopefully I didn't forget any):

- Which cycler
- Which frequency or maybe a burst for a short time
- Which game state
- Whether static framework.json, Communication, or robotics domain requested it

Reply:
Then I would suggest for now only enabling the recording during standby but not recording the `ObjectDetectionTop` cycler by default or by use of pregame. If we really need to record it, we can still use the `pepsi pregame --recording-intervals`.